rust/xaynet-core/src/mask/masking.rs

```diff
@@ -76,10 +94,10 @@ impl Into<MaskObject> for Aggregation {
 #[allow(clippy::len_without_is_empty)]
 impl Aggregation {
     /// Creates a new, empty aggregator for masks or masked models.
-    pub fn new(config: MaskConfig, object_size: usize) -> Self {
+    pub fn new(config_many: MaskConfig, config_one: MaskConfig, object_size: usize) -> Self {
```

Comment:
Not sure if it's worth it but maybe we could have a 

```rust
struct AggregationConfig {
  model: MaskConfig,
  scalar: MaskConfig,
}
```

Reply:
I think this would further increase tidiness, thanks. will add this in a later PR if that's ok.

Reply:
So this could be tackled with https://github.com/xaynetwork/xaynet/pull/524#discussion_r490782543 right?